CODE_OF_CONDUCT.md

```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-engineering@permanent.org.
+engineers@permanent.org.
```

Comment:
Unrelated to this PR but just a note for the team that we may want to just have the COC in the `.github` repo rather than duplicated so we don't have to update it in more than one repository.